containers/ddev-webserver/tests/ddev-webserver/testdata/nginx/auth.conf

```
@@ -0,0 +1,2 @@
+auth_basic   "auth test DDEV";
```

Lets use the same value here as for apache
```suggestion
auth_basic   "Restricted Files";
```